examples/redis/blueprint-v2/redis-blueprint.yaml

```diff
@@ -0,0 +1,86 @@
+apiVersion: cr.kanister.io/v1alpha1
+kind: Blueprint
```

Comment:
do we need this version of the blueprint? I don't think we should keep this if we are not testing this anywhere.

Reply:
@viveksinghggits didn't understand the question. are you suggesting to remove `v2` blueprint?

Reply:
removed v2 blueprint